Sources/MenuAnimator.swift

```diff
@@ -201,11 +238,14 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
         if present {
             containerView.insertSubview(toViewController.view, belowSubview: fromViewController.view)
 
-            self.tapRecognizer = UITapGestureRecognizer(target: toViewController as! MenuViewController,
+            if self.tapRecognizer == nil {
+
+                self.tapRecognizer = UITapGestureRecognizer(target: toViewController as! MenuViewController,
```

Comment:
it would be better replace ```toViewController as! MenuViewController``` to

```
guard let menuController = toViewController as? MenuViewController else {
    preconditionFailure or  fatalError
}
```